common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1432,48 +1432,102 @@ object CelebornConf extends Logging {
   val NETWORK_IO_PREFER_DIRECT_BUFS: ConfigEntry[Boolean] =
     buildConf("celeborn.<module>.io.preferDirectBufs")
       .categories("network")
-      .doc("If true, we will prefer allocating off-heap byte buffers within Netty.")
+      .doc("If true, we will prefer allocating off-heap byte buffers within Netty. " +
+        s"If setting <module> to `${TransportModuleConstants.RPC_MODULE}`, " +
+        s"it works for shuffle client, master or worker server rpc and should be configured on client side. " +
```

Comment:
```suggestion
        s"it works for shuffle client, master and worker. " +
```
I think works for shuffle client is enough for a user to understand that this config needs to be added to client.